lib/libzrepl/mgmt_conn.c

```diff
@@ -1237,6 +1237,7 @@ handle_start_rebuild_req(uzfs_mgmt_conn_t *conn, zvol_io_hdr_t *hdrp,
 		uzfs_update_ionum_interval(zinfo, 0);
 		LOG_INFO("Rebuild of zvol %s completed",
 		    zinfo->name);
+		quiesce_wait(zinfo, 1);
```

Comment:
in single replica case also, we need to rebuild from clone before destroying the clone. So, we would need to:
```
set_rebuild_status to AFS with lock
quiesce_wait(zinfo, 0)
start_rebuild_from_clone
once that is done, it will take care of doing quiesce_wait(zinfo, 1) to delete clone
```

Reply:
Done